examples/compose/vttablet-up.sh

```diff
@@ -108,58 +108,57 @@ sleep $sleeptime
 
 # Create the cell
 # https://vitess.io/blog/2020-04-27-life-of-a-cluster/
-$VTROOT/bin/vtctlclient -server vtctld:$GRPC_PORT AddCellInfo -root vitess/$CELL -server_address consul1:8500 $CELL || true
+$VTROOT/bin/vtctlclient --server vtctld:$GRPC_PORT AddCellInfo --root vitess/$CELL --server_address consul1:8500 $CELL || true
```

Comment:
I _believe_ you want an extra double dash before `AddCellInfo` (I think this prints out [this warning](https://github.com/vitessio/vitess/blob/main/go/internal/flag/flag.go#L92-L94))

Reply:
Yeah:
```
$ vtctlclient --server localhost:15999 AddCellInfo --root zone2 --server_address foo:1
W0513 15:48:13.814964   23983 flag.go:123] Detected a dashed argument after a positional argument. Currently these are treated as posargs that may be parsed by a subcommand, but in the next version of Vitess they will be parsed as top-level flags, which may not be defined, causing errors. To preserve existing behavior, please update your invocation to include a "--" after all top-level flags to continue treating --root as a positional argument.
...
```